docs/metrics.md

```diff
@@ -0,0 +1,142 @@
+
+# Tab Split Metrics
+
+Metrics collections and analysis plan for Tab Split as part of the [Firefox Test Pilot program](https://testpilot.firefox.com).
+
+## Analysis
+
+Data collected in this experiment will be used to answer the following high-level questions:
+
+* How often do users load a non-homepage in the sidebar?
+* Are people who use Tab Split heavy tab users?
+
+
+
+## Collection
+Data will be collected with Google Analytics and follow [Test Pilot standards](https://github.com/mozilla/testpilot/blob/master/docs/experiments/ga.md) for reporting.
+
+## Custom Metrics
+*none*
+
+### Custom Dimensions
+
+* `cd1` - The width of the sidebar in pixels.  An integer
+* `cd2` - The number of open tabs in the current window.  An integer
+* `cd3` - The site type requested.  One of `desktop` or `mobile`
+
+### Events
+
+#### `Interface`
+
+##### When the user chooses to load a link in the sidebar from the context menu
+```
+ec: interface,
+ea: load-url,
+el: context-menu,
+cd1,
+cd2,
+cd3
+```
+
+##### When the user chooses to load a link in the sidebar by choosing an existing tab on the home page
+```
+ec: interface,
+ea: load-url,
+el: existing-tab,
+cd1,
+cd2,
+cd3
+```
+
+##### When the user chooses to load a link in the sidebar by choosing a previously loaded tab on the home page
+```
+ec: interface,
+ea: load-url,
+el: previous-tab,
+cd1,
+cd2,
+cd3
+```
+
+##### When the user activates the Tab Split back button
+```
+ec: interface,
+ea: button-click,
+el: button-back,
+cd1
+```
+
+##### When the user activates the Tab Split refresh button
+```
+ec: interface,
+ea: button-click,
+el: button-refresh,
+cd1,
+cd3
+```
+
+##### When the user activates Tab Split from the Sidebar dropdown menu
```

Comment:
I'm not sure we can access this from a webextension.  If not, we should just drop it.

Reply:
There aren't many [sidebarAction events](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/sidebarAction), including this one. We can see when the sidebar opens, because it loads, but not necessarily *why*. I'm not sure if it would be confusing to just have a generic sidebar open event, that may or may not have an accompanying action...?

Reply:
I'll whack it